src/hotspot/share/utilities/enumIterator.hpp

```diff
@@ -229,6 +236,14 @@ class EnumRange {
     assert(size() > 0, "empty range");
   }
 
+private:
+
+  struct ConstExprConstructTag {};
```

Comment:
The above is implicitly private so this should probably be removed:
```suggestion
  struct ConstExprConstructTag {};
```